integration-tests/test/stress-tests.spec.ts

```diff
@@ -17,22 +17,26 @@ import {
 
 /* Imports: Artifacts */
 import simpleStorageJson from '../artifacts/contracts/SimpleStorage.sol/SimpleStorage.json'
-import { fundUser, isLiveNetwork } from './shared/utils'
+import { fundUser, isLiveNetwork, isMainnet } from './shared/utils'
 
 // Need a big timeout to allow for all transactions to be processed.
 // For some reason I can't figure out how to set the timeout on a per-suite basis
 // so I'm instead setting it for every test.
 const STRESS_TEST_TIMEOUT = isLiveNetwork() ? 500_000 : 1_200_000
 
-describe('stress tests', () => {
+describe.skip('stress tests', () => {
   const numTransactions = 3
 
   let env: OptimismEnv
 
   const wallets: Wallet[] = []
 
-  before(async () => {
+  before(async function () {
     env = await OptimismEnv.new()
+    if (await isMainnet(env)) {
+      this.skip()
```

Comment:
Perhaps not a bad idea to log the fact that its being skipped because of mainnet